mlflow/store/tracking/dbmodels/models.py

```diff
@@ -365,7 +365,7 @@ class SqlLatestMetric(Base):
     """
     Step recorded for this metric entry: `BigInteger`.
```

Comment:
@harupy are we also missing `create_constraint=True` for https://github.com/mlflow/mlflow/blob/df2d73e9167a6be93ec903c1be333221d0b2ec81/mlflow/store/tracking/dbmodels/models.py#L311? (`SqlMetric`)

Reply:
If so, we should also account for this in your migration above (https://github.com/mlflow/mlflow/pull/4180/files#r596167087).

Reply:
https://github.com/mlflow/mlflow/blob/8e74cdd108ff324175e38d621b12d9b2aced05bd/mlflow/store/db_migrations/versions/181f10493468_allow_nulls_for_metric_values.py#L23

Thanks for the catch but I found we set `create_constraint` to `False` for `is_nan` in the `metrics` table.

Reply:
This is definitely a bit strange! For consistency, can we add an `alter_column` with `create_constraint=True` for the migration you're introducing?